packages/core/src/browser/keybinding.ts

```diff
@@ -652,6 +652,7 @@ export class KeybindingRegistry {
     setKeymap(scope: KeybindingScope, bindings: Keybinding[]): void {
         this.resetKeybindingsForScope(scope);
         this.doRegisterKeybindings(bindings, scope);
+        this.keybindingsChanged.fire(undefined);
```

Comment:
Currently `onKeybindingsChanged` is documented as

> Event that is fired when the resolved keybindings change due to a different keyboard layout.

By firing that event in other circumstances, you should also update that comment.

Reply:
@spoenemann Fixed